services/QuillLMS/app/assets/stylesheets/shared/modals.scss

```diff
@@ -449,6 +449,9 @@
       line-height: 1.57;
     }
   }
+  &:focus {
+    outline: none;
+  }
```

Comment:
Why do we want no outline here?

Reply:
do we want the entire modal to be outlined? I figured we just wanted the first button element in a modal to be focusable

Reply:
Oh, I was confused about what that applied to. Yes, that looks right.